examples/example-imgui.hpp

```diff
@@ -47,10 +47,28 @@ class hdr_slider : public slider {
         ImGui::SetNextWindowPos(_position);
         //concate the name given with seq_id in order to make a unique name (uniqeness is needed for Begin())
         std::string name_id = std::string(_name) + std::to_string(_seq_id);
+        ImGuiStyle& style = ImGui::GetStyle();
+        ImVec4* colors = style.Colors;
```

Comment:
Why do we need all this color settings that was not done before?

Reply:
It was done as part of the imgui.cpp (changed the style and color of the base tools of imgui) 
In the new version the base colors are blue and black and in the example-imgui.hpp it looked different, so I added the style and colors